modules/gke-nodepool/variables.tf

```diff
@@ -64,12 +64,17 @@ variable "node_config" {
     ephemeral_ssd_count = optional(number)
     gcfs                = optional(bool, false)
     guest_accelerator = optional(object({
-      count              = number
-      type               = string
-      gpu_partition_size = optional(string)
+      count = number
+      type  = string
+      gpu_driver = optional(object({
+        version                    = string
```

Comment:
The allowed values for type are
- GPU_DRIVER_VERSION_UNSPECIFIED
- INSTALLATION_DISABLED
- DEFAULT
- LATEST

Can you add a validation similar to [this one](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/modules/billing-account/variables.tf#L36)?